footprints/main/views.py

```diff
@@ -193,13 +193,16 @@ def get_context_data(self, **kwargs):
             self.request.GET.get('footprint_start_year') or
             self.request.GET.get('footprint_end_year'))
 
+        context['gallery_view'] = self.request.GET.get('gallery_view')
```

Comment:
I generally provide a default value for this type of initialization. And, nicely, the `get` function offers an easy way to do this:
`self.request.GET.get('gallery_view', False)`